redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/start/actions/CertRotateInRedbeamsAction.java

```diff
@@ -0,0 +1,21 @@
+package com.sequenceiq.redbeams.flow.redbeams.start.actions;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.redbeams.flow.redbeams.common.RedbeamsContext;
+import com.sequenceiq.redbeams.flow.redbeams.start.event.CertRotateInRedbeamsRequest;
+import com.sequenceiq.redbeams.flow.redbeams.start.event.StartDatabaseServerSuccess;
+
+@Component("CERT_ROTATE_STATE")
+public class CertRotateInRedbeamsAction extends AbstractRedbeamsStartAction<StartDatabaseServerSuccess> {
```

Comment:
Please add UT for this action

Reply:
@topolyai5 could you please also add a UT here for checking if createRequest returns the correct event?